javascript/javascript1/week4/command.js

```diff
@@ -0,0 +1,75 @@
+//voice assistant 🤖!
+
+let inPutName = "";
+let toDoList = [];
+function getReply(command){
+   
+      let inputValue = command.split(" ") 
+      if (command.includes("Hello my name is ")) {
+        if ( inPutName===  inputValue[ inputValue.length - 1]) {
+            return `Your are already registered.`
+        } else {
+            inPutName = inputValue[ inputValue.length - 1];
+            return `Nice to meet you ${inPutName}.`;
+        }
+          }  if (command === "What is my name") {
+             if (!inPutName) {
+             return `your name is not registered yet`;
+        }   else {
+            return `Your name is ${inPutName}.`;
+        }
+    }
+          if (command === "Add fishing to my todo") {
+          toDoList.push("Fishing");
+          return `Fishing added to my todo`;
+    }
```

Comment:
Could you make it so that any item could be added to your todo? For example, Add cooking dinner to my todo.

Reply:
Yes, my code was unable to do that but I try to change it.  Now it is working